packages/lesswrong/components/ea-forum/EASequenceOrCollectionCard.tsx

```diff
@@ -0,0 +1,143 @@
+import React, { MouseEvent } from "react";
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import { InteractionWrapper, useClickableCell } from "../common/useClickableCell";
+import { Link } from "../../lib/reactRouterWrapper";
+
+const SEQUENCE_CARD_IMAGE_HEIGHT = 162;
+const Z_IMAGE = 1;
+const Z_OVERLAY = 2;
+
+const buildBoxShadow = (theme: ThemeType, alpha: number, offset: number) => `
+  /* The top layer shadow */
+  0px ${offset}px 6px 0px ${theme.palette.boxShadowColor(alpha)},
+  /* The second layer */
+  -8px 8px 0 0px ${theme.palette.panelBackground.default},
+  /* The second layer shadow */
+  -10px 10px 10px -1px ${theme.palette.boxShadowColor(alpha)}
+`;
+
+const styles = (theme: ThemeType) => ({
+  root: {
+    cursor: "pointer",
+    borderRadius: theme.borderRadius.default,
+    background: theme.palette.panelBackground.default,
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontWeight: 500,
+    marginLeft: 8, // Account for box shadow
+    marginBottom: 8, // Account for box shadow
+    boxShadow: buildBoxShadow(theme, 0.1, 2),
+    transition: "box-shadow 0.15s ease-in-out",
+    "&:hover": {
+      boxShadow: buildBoxShadow(theme, 0.15, 4),
+    },
+  },
+  sequenceCardImageWrapper: {
+    position: "relative",
+  },
+  sequenceCardImage: {
+    width: "100%",
+    height: SEQUENCE_CARD_IMAGE_HEIGHT,
+    objectFit: "cover",
+    borderTopLeftRadius: theme.borderRadius.default,
+    borderTopRightRadius: theme.borderRadius.default,
+    zIndex: Z_IMAGE,
+  },
+  sequenceReadProgress: {
+    position: "absolute",
+    zIndex: Z_OVERLAY,
+    fontSize: 12,
+    top: 8,
+    left: 8,
+    borderRadius: 14,
+    backgroundColor: theme.palette.panelBackground.translucent3,
+    padding: "6px 8px",
+  },
+  sequenceCardText: {
+    padding: 16,
+  },
+  sequenceCardTitle: {
+    fontSize: 18,
+    lineHeight: "24px",
+    overflow: "hidden",
+    display: "-webkit-box",
+    "-webkit-box-orient": "vertical",
+    "-webkit-line-clamp": 2,
+    marginBottom: 6,
+  },
+  sequenceCardMeta: {
+    fontSize: 14,
+    lineHeight: "20px",
+    color: theme.palette.grey[600],
+  },
+  authorWrapper: {
+    display: "inline",
+  },
+});
+
+const EASequenceOrCollectionCard = ({
```

Comment:
In the initial version, part of the reason I did these components inline was because they overlap with CollectionsCard, BigCollectionsCard, and SequencesGridItem in the library page, and I wasn't sure if we wanted to fully combine. We now do want to do that so I think it would be good to combine all of these into one (so the components in the library page use this styling)

Sorry I should have communicated this, but I do think this is still worth doing even for this PR, and that it shouldn't be that difficult(?) It may require some EACollectionsCard/LWCollectionsCard style routing

Reply:
Makes sense - I'll save it for a separate PR though